### PR TITLE
fixing spelling and grammar in db md

### PR DESCRIPTION
**Pull request by james-francis-MT**
Small spelling and grammar changes to the database.md file

### CODE REVIEW

File: modules/supplementary/Databases.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 ## Relational Databases ![mysqldownload](https://user-images.githubusercontent.com/114578618/195858167-b4ae6082-d376-44f2-951e-f762d1d07594.png)![ms3download](https://user-images.githubusercontent.com/114578618/195859896-602ebada-d592-4421-8558-b38c4f4f6e9b.png)![pgsdownload](https://user-images.githubusercontent.com/114578618/195858073-de817956-6540-423b-ad0c-d0c9f196539d.png)
 
-Databases are efficient ways of storing and managing data. Databases store organised or structured data using a relational model which stores data in two dimensional format like tables, represented in rows and columns
+Databases are efficient ways of storing and managing data. Databases store organised or structured data using a relational model which stores data in a two dimensional format like tables, represented in rows and columns
 
 Each row in a table represents a single record with unique ID called Key. A record is a meaningful and consistent way of combining information about something.  
-A column represents a field of attribute or an item of information. 
+A column represents a field or attribute of an item of information.
```
Comment (james-francis-MT) [inline modules/supplementary/Databases.md:8]: Is this correct?

File: modules/supplementary/Databases.md [hunk 2]
```diff
@@ -402,9 +393,9 @@ Data replication may not be instantaneous as some copies will be updated immedia
 
   -	The concept of NoSQL databases became popular with Internet giants like Google, Facebook, Amazon, etc. who deal with huge volumes of data
 
-  -	In the year 1998- Carlo Strozzi use the term NoSQL for his lightweight, open-source relational database
+  -	In the year 1998- Carlo Strozzi used the term NoSQL for his lightweight, open-source relational database
```
Comment (james-francis-MT) [inline modules/supplementary/Databases.md:396]: Coined?

File: modules/supplementary/Databases.md (hunk 1 above)
Comment (abiadekunle) [reply modules/supplementary/Databases.md:8]: sounds good

File: modules/supplementary/Databases.md (hunk 2 above)
Comment (abiadekunle) [reply modules/supplementary/Databases.md:396]: yeah good spot :)

**Review by abiadekunle — APPROVED** [review]

**Pull request merged**